src/config/configuration.ts

```diff
@@ -26,16 +26,16 @@ export interface ConfigurationProps {
 export interface Configuration {
   // TODO: add Middlewares
```

Comment:
Is this TODO affected by the pr? I don't know what middleware means here.

Reply:
nope good question though!  in the .net SDK we have a middleware interface.  we don't have it here yet.  i'm not sure whether i will try to get that in before 1.0 or not.